src/components/comments/CommentSection.js

```diff
@@ -20,8 +20,7 @@ a corresponding question or answer*/
   /* Only rendering the ones that should be visible at that current moment*/
   let visibleFollowups = allFollowups.slice(0, props.currentVisibleText);
   return (
-    <div className="comments-container content-container ">
-      
+    <div className="comments-container content-container " id={props.followups[0].question}>
```

Comment:
Quick q: just to double-check, does this mean that each followup's first item must be a question?

Reply:
Yeah! The purpose of having a followup section is to provide the student with questions related to the section. The _answers_ to each question are optional, and always come after the question.

Reply:
Ah gotcha gotcha. For now this is good then, I think eventually we should document that relationship (or ... enforce it with typescript 😉 )